includes/ziti/ziti.h

```diff
@@ -330,10 +336,11 @@ typedef void (*ziti_conn_cb)(ziti_connection conn, int status);
  * @param serv hosting connection, initialized with ziti_listen()
  * @param client client connection - generally passed to ziti_accept() in this function
  * @param status #ZITI_OK or error
+ * @param ctx object containing application data passed by dialing identity, \see ziti_dial_opts
```

Comment:
it may be obvious, but should this mention that the contents of `ctx` are only valid in the client_cb?

Reply:
done